diffs/auditbeat2.diff

```diff
@@ -0,0 +1,1923 @@
+html/en/beats/auditbeat/master/add-tags.html
+html/en/beats/auditbeat/master/auditbeat-configuration-reloading.html
+html/en/beats/auditbeat/master/auditbeat-configuration.html
+--- /dev/fd/63
++++ html/en/beats/auditbeat/master/auditbeat-configuration.html
+@@ -439,17 +439,17 @@
+                      </a>
+                      or equivalent privileges.
+                     </p>
+-                    <p>
+-                     For more information, see
+-                     <a class="xref" href="securing-auditbeat.html" title="Securing Auditbeat">
+-                      Securing Auditbeat
+-                     </a>
+-                     .
+-                    </p>
+                    </td>
+                   </tr>
+                  </table>
+                 </div>
++                <p class="simpara">
++                 For more information, see
++                 <a class="xref" href="securing-auditbeat.html" title="Securing Auditbeat">
++                  Securing Auditbeat
++                 </a>
++                 .
++                </p>
+                </li>
+               </ul>
+              </div>
+html/en/beats/auditbeat/master/auditbeat-dataset-system-host.html
+html/en/beats/auditbeat/master/auditbeat-dataset-system-login.html
+html/en/beats/auditbeat/master/auditbeat-dataset-system-package.html
+html/en/beats/auditbeat/master/auditbeat-dataset-system-process.html
+html/en/beats/auditbeat/master/auditbeat-dataset-system-socket.html
+html/en/beats/auditbeat/master/auditbeat-dataset-system-user.html
+html/en/beats/auditbeat/master/auditbeat-geoip.html
+--- /dev/fd/63
++++ html/en/beats/auditbeat/master/auditbeat-geoip.html
+@@ -226,8 +226,8 @@
+               </code>
+               processors to add location information to the event. For example, you can use the Console in Kibana to create the following pipeline:
+              </p>
+-             <div class="pre_wrapper lang-json">
+-              <pre class="programlisting prettyprint lang-json">PUT _ingest/pipeline/geoip-info
++             <div class="pre_wrapper lang-console">
++              <pre class="programlisting prettyprint lang-console">PUT _ingest/pipeline/geoip-info
+ {
+   "description": "Add geoip info",
+   "processors": [
+html/en/beats/auditbeat/master/auditbeat-getting-started.html
+html/en/beats/auditbeat/master/auditbeat-installation.html
+html/en/beats/auditbeat/master/auditbeat-module-auditd.html
+html/en/beats/auditbeat/master/auditbeat-module-file_integrity.html
+html/en/beats/auditbeat/master/auditbeat-module-system.html
+html/en/beats/auditbeat/master/auditbeat-modules.html
+--- /dev/fd/63
++++ html/en/beats/auditbeat/master/auditbeat-modules.html
+@@ -161,6 +161,9 @@
+            <p>
+             This section contains detailed information about the metric collecting modules contained in Auditbeat. More details about each module can be found under the links below.
+            </p>
++           <p>
++            pass::[&lt;?edit_url?&gt;]

```

Comment:
fix required

Reply:
fixed in https://github.com/elastic/beats/commit/a11ca121ee73c33767e4f2d2143debeaf52962ea